Hush/Assets/Animations/Door Text.controller

```diff
@@ -0,0 +1,213 @@
+%YAML 1.1
```

Comment:
File name has double space

Reply:
deleted in [this](https://github.com/Unlimited-Pasta-Pass/Hush-game/pull/43/commits/a5de78f4f25c560939be511ccbd71f737cbeb3bb) commit because I removed the animation 👍